tsconfig-dist.json

```diff
@@ -0,0 +1,36 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "es2015",
```

Comment:
@BrainCrumbz unlike webpack 2 webpack 1 **does not support** es6 imports. This is the main difference between v1 and v2. See https://webpack.github.io/docs/?utm_source=github&utm_medium=readme&utm_campaign=top 

> webpack supports AMD and CommonJs module styles